server/server.js

```diff
@@ -1,18 +1,41 @@
 const express = require('express');
-const app = express(); 
+
+const app = express();
 
 const bodyParser = require('body-parser');
-const path = require("path");
-const axios = require('axios')
+const path = require('path');
+
+const pgp = require('pg-promise')();
+
+const config = {
+  host: 'localhost',
+  user: 'mrmac', // env var: PGUSER
+  database: 'shamazon_main_description', // env var: PGDATABASE
+  password: null, // env var: PGPASSWORD
+  port: 5000, // env var: PGPORT
+};
 
+const db = pgp(config);
 
+app.get('/products/:productkey', (req, res) => {
```

Comment:
Rather than using productkey, do you think it'd be better to use something along the lines of /products/:id/description as Shi-Hao suggested? It'd also make for more uniformity as the rest of the group is going by /products/:id/xyz.